app/renderer/components/common/messageBox.js

```diff
@@ -3,20 +3,34 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const React = require('react')
-const ImmutableComponent = require('../immutableComponent')
+const {StyleSheet, css} = require('aphrodite')
```

Comment:
@luixxiul maybe? we could do that, but it would require testing. I would vote to leave it as is.  IMO, using `no-important` takes away the benefit we get by using Aphrodite.

We introduced Aphrodite to our code base because we were getting destroyed by regressions and the goal (I thought) was to make styles NOT overridable.  When it's `no-important`, styles might be overridden (which can cause regression).